<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1670</width>
    <height>920</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Tahoma</family>
    <pointsize>10</pointsize>
    <italic>true</italic>
   </font>
  </property>
  <property name="windowTitle">
   <string>Computer Aided Diagnosis</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #1A1A1A;

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>680</y>
      <width>160</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 6px; 
    background-color: #14A37B; 
    color: #F5F5F5; /* Warna teks */
    padding: 8px 16px; /* Atur padding tombol */
}

QPushButton:hover {
    background-color: #137856; /* Warna latar belakang saat hover */
}

QPushButton:pressed {
    background-color: #145E4E; /* Warna latar belakang saat tombol ditekan */
}</string>
    </property>
    <property name="text">
     <string>Input Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>680</y>
      <width>160</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 6px; 
    background-color: #14A37B; 
    color: #F5F5F5; /* Warna teks */
    padding: 8px 16px; /* Atur padding tombol */
}

QPushButton:hover {
    background-color: #137856; /* Warna latar belakang saat hover */
}

QPushButton:pressed {
    background-color: #145E4E; /* Warna latar belakang saat tombol ditekan */
}</string>
    </property>
    <property name="text">
     <string>Enhancement</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>680</y>
      <width>160</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 6px; 
    background-color: #14A37B; 
    color: #F5F5F5; /* Warna teks */
    padding: 8px 16px; /* Atur padding tombol */
}

QPushButton:hover {
    background-color: #137856; /* Warna latar belakang saat hover */
}

QPushButton:pressed {
    background-color: #145E4E; /* Warna latar belakang saat tombol ditekan */
}</string>
    </property>
    <property name="text">
     <string>Draw Contours</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>680</y>
      <width>160</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 6px; 
    background-color: #14A37B; 
    color: #F5F5F5; /* Warna teks */
    padding: 8px 16px; /* Atur padding tombol */
}

QPushButton:hover {
    background-color: #137856; /* Warna latar belakang saat hover */
}

QPushButton:pressed {
    background-color: #145E4E; /* Warna latar belakang saat tombol ditekan */
}</string>
    </property>
    <property name="text">
     <string>Segmentation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>1470</x>
      <y>680</y>
      <width>160</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 6px; 
    background-color: #14A37B; 
    color: #F5F5F5; /* Warna teks */
    padding: 8px 16px; /* Atur padding tombol */
}

QPushButton:hover {
    background-color: #137856; /* Warna latar belakang saat hover */
}

QPushButton:pressed {
    background-color: #145E4E; /* Warna latar belakang saat tombol ditekan */
}</string>
    </property>
    <property name="text">
     <string>Extraction</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #F5F5F5;
background-color: transparent</string>
    </property>
    <property name="text">
     <string>Preprocessing Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>110</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #F5F5F5;
background-color: transparent</string>
    </property>
    <property name="text">
     <string>Segmentation Image &amp; Abnormal Cell Plot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>1470</x>
      <y>830</y>
      <width>160</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 6px; 
    background-color: #14A37B; 
    color: #F5F5F5; /* Warna teks */
    padding: 8px 16px; /* Atur padding tombol */
}

QPushButton:hover {
    background-color: #137856; /* Warna latar belakang saat hover */
}

QPushButton:pressed {
    background-color: #145E4E; /* Warna latar belakang saat tombol ditekan */
}</string>
    </property>
    <property name="text">
     <string>Save Diagnosis</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>1470</x>
      <y>770</y>
      <width>160</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 6px; 
    background-color: #E5172F; 
    color: #F5F5F5; /* Warna teks */
    padding: 8px 16px; /* Atur padding tombol */
}

QPushButton:hover {
    background-color: #C9162B; /* Warna latar belakang saat hover */
}

QPushButton:pressed {
    background-color: #85101E; /* Warna latar belakang saat tombol ditekan */
}</string>
    </property>
    <property name="text">
     <string>Clear Process</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>160</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_3">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>160</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1653</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 6px; /* Ubah angka sesuai dengan tingkat bulatan yang diinginkan */
background-color: #14A37B;
color: #F5F5F5; /* Warna teks */
padding: 8px 16px; /* Atur padding tombol */
</string>
    </property>
    <property name="text">
     <string>Lung Tumour Diagnosis</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>110</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #F5F5F5;
background-color: transparent</string>
    </property>
    <property name="text">
     <string>Feature Extraction</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>780</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #F5F5F5;
background-color: transparent;</string>
    </property>
    <property name="text">
     <string>Lung Detection:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>810</y>
      <width>160</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 6px; 
    background-color: #14A37B; 
    color: #F5F5F5; /* Warna teks */
    padding: 8px 16px; /* Atur padding tombol */
}

QPushButton:hover {
    background-color: #137856; /* Warna latar belakang saat hover */
}

QPushButton:pressed {
    background-color: #145E4E; /* Warna latar belakang saat tombol ditekan */
}</string>
    </property>
    <property name="text">
     <string>Detection</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0); /* Putih dengan 0% opasitas (transparansi) */
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>asset/Logo_PENS_putih.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>750</y>
      <width>1431</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #F5F5F5;
background-color:transparent;</string>
    </property>
    <property name="title">
     <string>Classification Image</string>
    </property>
    <widget class="QPlainTextEdit" name="result_text">
     <property name="geometry">
      <rect>
       <x>1000</x>
       <y>70</y>
       <width>380</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPlainTextEdit {
    border-radius: 4px; /* Atur radius sudut sesuai keinginan Anda */
    border: 1px solid #ccc; /* Atur garis pinggiran jika diinginkan */
	color: #F5F5F5;
}</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>1001</x>
       <y>30</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #F5F5F5;
background-color: transparent;</string>
     </property>
     <property name="text">
      <string>Diagnosis Result:</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="cell_text">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>70</y>
       <width>260</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPlainTextEdit {
    border-radius: 4px; /* Atur radius sudut sesuai keinginan Anda */
    border: 1px solid #ccc; /* Atur garis pinggiran jika diinginkan */
	color: #F5F5F5;
}</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="backgroundVisible">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>30</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #F5F5F5;
background-color: transparent;</string>
     </property>
     <property name="text">
      <string>Abnormal Cell:</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="detect_text">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>70</y>
       <width>260</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPlainTextEdit {
    border-radius: 4px; /* Atur radius sudut sesuai keinginan Anda */
    border: 1px solid #ccc; /* Atur garis pinggiran jika diinginkan */
	color: #F5F5F5;
}</string>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string/>
     </property>
     <property name="backgroundVisible">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>60</y>
       <width>160</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border-radius: 6px; 
    background-color: #14A37B; 
    color: #F5F5F5; /* Warna teks */
    padding: 8px 16px; /* Atur padding tombol */
}

QPushButton:hover {
    background-color: #137856; /* Warna latar belakang saat hover */
}

QPushButton:pressed {
    background-color: #145E4E; /* Warna latar belakang saat tombol ditekan */
}</string>
     </property>
     <property name="text">
      <string>Diagnosis</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>60</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #F5F5F5;
background-color: transparent</string>
    </property>
    <property name="text">
     <string>Created by Latif @ 2024</string>
    </property>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>graphicsView</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>pushButton_9</zorder>
   <zorder>pushButton_8</zorder>
   <zorder>graphicsView_2</zorder>
   <zorder>graphicsView_3</zorder>
   <zorder>label_9</zorder>
   <zorder>label_10</zorder>
   <zorder>label_5</zorder>
   <zorder>pushButton_7</zorder>
   <zorder>label</zorder>
   <zorder>label_11</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1670</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
